apps/system/test/unit/wifi_test.js

```diff
@@ -538,8 +507,8 @@ suite('WiFi > ', function() {
       Wifi.wifiEnabled = false;
       navigator.battery = { charging: false };
 
-      var isWifiPowerSavingMode = false;
-      var isSetPowerSavingModeCalled = false;
+      var isWifiPowerSavingMode = true;
+      var isSetPowerSavingModeCalled = true;
```

Comment:
I think initialize as true and check if the value is true later is not a good idear. Need to consider if the test case should be kept or modified properly.